src/python/pants/backend/project_info/tasks/filedeps.py

```diff
@@ -16,18 +16,23 @@
 class FileDeps(ConsoleTask):
   """List all source and BUILD files a target transitively depends on.
 
-  Files are listed with absolute paths and any BUILD files implied in the transitive closure of
-  targets are also included.
+  Files may be listed with absolute or relative paths and any BUILD files implied in the transitive
+  closure of targets are also included.
   """
 
   @classmethod
   def register_options(cls, register):
     super(FileDeps, cls).register_options(register)
     register('--globs', type=bool,
              help='Instead of outputting filenames, output globs (ignoring excludes)')
+    register('--abs', type=bool, default=True,
+             help='If True output with absolute path, else output with relative path')
 
-  def _full_path(self, path):
-    return os.path.join(get_buildroot(), path)
+  def _files_path(self, path):
+    abs_path = self.get_options().abs
```

Comment:
Let's inline this variable - `abs_path` implies to me that it's a path, rather than a boolean, but:
```python
if self.get_options().absolute:
```
reads pretty well to me :)